<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GererLesRep</class>
 <widget class="QDialog" name="GererLesRep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gérer les représentations</string>
  </property>
  <widget class="QComboBox" name="comboBox_spectacle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Sélectionner un spectacle</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>391</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableTicketsRep">
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>64</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>noAchat</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>noTicket</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>noRang</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>noPlace</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>reduction</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>prix</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>688</x>
     <y>20</y>
     <width>311</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableWidget" name="tableTicketsToAdd">
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>57</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>catZone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>noRang</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>noPlace</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>reduction</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>prix</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox_confirmAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>766</x>
     <y>440</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_nbTicket">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>90</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_nbTicket">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>89</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de tickets</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_typeReduc">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>130</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_typeReduc">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>132</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type de réduction</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_rep">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Filtrer une date de représentation</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_resTicket">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;Réservation ticket&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_zone">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>172</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zone</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_zone">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>170</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_prixAvecReduction">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>338</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Prix avec réduction :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_montantAvecReduc">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>338</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_formTicket">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addTicket">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter à a file de création</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deleteTicket">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_aucunSpec">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#797979;&quot;&gt;Aucun spectacle sélectionné&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_aucunTicket">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#797979;&quot;&gt;Aucun ticket à afficher&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_feedback">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_prix">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>275</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Montant :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_promoRep">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>305</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Promo représentation :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_montant">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>275</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_valeurPromoRep">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>305</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_symEuro_3">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>275</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_symEuro_4">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>338</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_symPourcentage">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>305</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prix_spec">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>212</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Prix spectacle :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_symEuro_1">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>212</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_montant_spec">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>212</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_montant_ticket">
   <property name="geometry">
    <rect>
     <x>539</x>
     <y>243</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_prix_ticket">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>243</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Prix ticket :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_symEuro_2">
   <property name="geometry">
    <rect>
     <x>654</x>
     <y>243</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_symEuro_5">
   <property name="geometry">
    <rect>
     <x>654</x>
     <y>372</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prixTotal">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>373</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Total :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_montantTotal">
   <property name="geometry">
    <rect>
     <x>589</x>
     <y>372</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>366</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
